src/test/java/step_definitions/SalesForceMarketing.java

```diff
@@ -34,75 +35,105 @@ public void readTheTestdataFromExcelFile(String arg1, String arg2) {
     }
 
     @Given("user navigate to the salesforce application")
-    public void user_navigate_to_application(){
+    public void user_navigate_to_application() {
         KeywordUtil.navigateToUrl(dataMap.get("URL"));
     }
 
     @When("user login into the application")
-    public void login_into_application(){
-        LoginSalesforceUtil.loginToSalesforce(dataMap.get("Username"),dataMap.get("Password"));
+    public void login_into_application() {
+        LoginSalesforceUtil.loginToSalesforce(dataMap.get("Username"), dataMap.get("Password"));
     }
 
     @And("user clicks on the App launcher button")
-    public void user_clicks_on_app_launcher(){
+    public void user_clicks_on_app_launcher() {
         AccountUtil.clickSlideShowButton("user clicked on the App launcher");
 
     }
 
     @And("user selects the {string}")
     public void select_option(String option) {
-      CampaignUtil.selectOption(option);
+        CampaignUtil.selectOption(option);
     }
 
     @Then("user clicks on Campaigns Tab")
     public void clicks_Campaigns() throws InterruptedException {
-      CampaignUtil.clickCampaignTab("click on the compaign tab");
+        CampaignUtil.clickCampaignTab("click on the compaign tab");
     }
 
     @Then("user create new Campaign")
-    public void create_New_Compaign(){
-        try{
-            CampaignUtil.createCampaign(dataMap.get("ComapaignName"));
+    public void create_New_Compaign() {
+        try {
+            CampaignUtil.createCampaign(dataMap.get("CamapaignName"));
```

Comment:
Please check the spelling of the column Name 'CamapaignName'

Reply:
I have checked it

Reply:
Done